MicaForEveryone.App/Strings/ja-JP/Resources.resw

```diff
@@ -0,0 +1,315 @@
+							</xsd:sequence>
+							<xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1"/>
+							<xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3"/>
+							<xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4"/>
+							<xsd:attribute ref="xml:space"/>
+						</xsd:complexType>
+					</xsd:element>
+					<xsd:element name="resheader">
+						<xsd:complexType>
+							<xsd:sequence>
+								<xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1"/>
+							</xsd:sequence>
+							<xsd:attribute name="name" type="xsd:string" use="required"/>
+						</xsd:complexType>
+					</xsd:element>
+				</xsd:choice>
+			</xsd:complexType>
+		</xsd:element>
+	</xsd:schema>
+	<resheader name="resmimetype">
+		<value>text/microsoft-resx</value>
+	</resheader>
+	<resheader name="version">
+		<value>2.0</value>
+	</resheader>
+	<resheader name="reader">
+		<value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+	<resheader name="writer">
+		<value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+	<data name="AboutCard.Header" xml:space="preserve">
+		<value>Mica For Everyoneについて</value>
+	</data>
+	<data name="AboutLabel.Text" xml:space="preserve">
+		<value>情報</value>
+	</data>
+	<data name="AppSettingsPage.Header" xml:space="preserve">
+		<value>アプリの設定</value>
+	</data>
+	<data name="TitleBarControl.Title" xml:space="preserve">
+		<value>Mica For Everyone</value>
+	</data>
+	<data name="BackdropTypeCard.Header" xml:space="preserve">
+		<value>背景の種類</value>
+	</data>
+	<data name="BlurBehindCard.Header" xml:space="preserve">
+		<value>背景をぼかす</value>
+	</data>
+	<data name="CheckForUpdatesButton.Content" xml:space="preserve">
+		<value>更新を確認</value>
+	</data>
+	<data name="CheckForUpdatesCard.Header" xml:space="preserve">
+		<value>更新を確認</value>
+	</data>
+	<data name="CornerPreferenceCard.Header" xml:space="preserve">
+		<value>角の設定</value>
+	</data>
+	<data name="DangerousInfoBar.Title" xml:space="preserve">
+		<value>注意が必要なオプション</value>
+	</data>
+	<data name="DangerousInfoBar.Message" xml:space="preserve">
+		<value>以下の設定はアプリを壊す可能性があります。どれか有効にする前に、作業データの保存を忘れずに！</value>
+	</data>
+	<data name="ExitFlyoutItem.Text" xml:space="preserve">
+		<value>終了</value>
+	</data>
+	<data name="ExtendFrameCard.Header" xml:space="preserve">
+		<value>クライアント領域までフレームを拡張</value>
+	</data>
+	<data name="RemoveCard.Header" xml:space="preserve">
+		<value>ルールを削除</value>
+	</data>
+	<data name="RemoveLabel.Text" xml:space="preserve">
+		<value>削除</value>
+	</data>
+	<data name="SettingsFlyoutItem.Text" xml:space="preserve">
+		<value>設定</value>
+	</data>
+	<data name="StartupCard.Description" xml:space="preserve">
+		<value>起動時にMica For Everyonを実行 </value>

```

Comment:
Is this misspelled?

Reply:
Thanks. This is a typo. I have fixed it.